deps.ffmpeg/70-libdatachannel.zsh

```diff
@@ -0,0 +1,112 @@
+autoload -Uz log_debug log_error log_info log_status log_output
+
+## Dependency Information
+local name='libdatachannel'
+local version='0.19.0'
+local url='https://github.com/paullouisageneau/libdatachannel.git'
+local hash='004c6b74ffa1d2718499b84d5bd7674d94b83dbc'
+
+## Dependency Overrides
+local -i shared_libs=1
+local dir="${name}-${version}"
+
+## Build Steps
+setup() {
+  log_info "Setup (%F{3}${target}%f)"
+  setup_dep ${url} ${hash}
+}
+
+clean() {
+  cd "${dir}"
+
+  if [[ ${clean_build} -gt 0 && -d "build_${arch}" ]] {
+    log_info "Clean build directory (%F{3}${target}%f)"
+
+    rm -rf "build_${arch}"
+  }
+}
+
+config() {
+  autoload -Uz mkcd progress
+
+  local _onoff=(OFF ON)
+
+  args=(
+    ${cmake_flags}
+    -DENABLE_SHARED="${_onoff[(( shared_libs + 1 ))]}"
+    -DUSE_MBEDTLS=1
+    -DNO_WEBSOCKET=1
+    -DNO_TESTS=1
+    -DNO_EXAMPLES=1
+  )
+  case ${target} {
+    windows-x*)
+      args+=(
+        -DCMAKE_CXX_FLAGS="-static-libgcc -static-libstdc++ -w -pipe -fno-semantic-interposition -static"
+        -DCMAKE_C_FLAGS="-static-libgcc -w -pipe -fno-semantic-interposition -static"
+        -DCMAKE_SHARED_LINKER_FLAGS="-static-libgcc -static-libstdc++ -L${target_config[output_dir]}/lib -Wl,--exclude-libs,ALL -static"
+      )
```

Comment:
Add a toolchain override here:

`-DCMAKE_TOOLCHAIN_FILE="${target_config[posix_toolchain]}"`.

Reply:
Then, to enable this, add

`[posix_toolchain]="${funcsourcetrace[1]:A:h}/toolchain/windows-cross-toolchain-posix.cmake"`

to lines 49 and 57 in `30-build.zsh` to add it to the target config array.

Reply:
Done